PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/AdLoading/PBMInterstitialAdLoader.m

```diff
@@ -71,17 +69,17 @@ - (void)reportSuccessWithAdObject:(id)adObject adSize:(nullable NSValue *)adSize
         }];
         return;
     }
-    if ([adObject conformsToProtocol:@protocol(PBMInterstitialEventHandler)]) {
-        id<PBMInterstitialEventHandler> const eventHandler = (id<PBMInterstitialEventHandler>)adObject;
+    if ([adObject conformsToProtocol:@protocol(InterstitialEventHandlerProtocol)]) {
+        id<PBMInterstitialAd> const eventHandler = (id<PBMInterstitialAd>)adObject;
```

Comment:
Here we are checking confirmation to InterstitialEventHandlerProtocol but casting to PBMInterstitialAd. It's a little bit confusing. Let's cast to InterstitialEventHandlerProtocol.